.travis.yml

```diff
@@ -1,19 +1,17 @@
 sudo: required
+
 services:
 - docker
-language: node_js
-node_js:
-- node
+
 script:
-- npm run lint
-- npm run snyk-test
-- npm run generate-coverage
-- npm run check-coverage
-- npm run upload-coverage-coveralls
+  - docker build --build-arg NODE_ENV=development -t nhsuk/nearby-api .
+  - docker run -d --name mongo "nhsuk/mongodb-pharmacy:$MONGO_IMAGE_TAG"
+  - docker run -e TRAVIS=true -e TRAVIS_JOB_ID=$TRAVIS_JOB_ID -e TRAVIS_BRANCH=$TRAVIS_BRANCH -e COVERALLS_REPO_TOKEN=$COVERALLS_REPO_TOKEN -e SNYK_TOKEN=$SNYK_TOKEN -e TRAVIS_COMMIT=$TRAVIS_COMMIT --link mongo:mongo nhsuk/nearby-api sh -c "cd /code && npm run git-hook && npm run snyk-test && npm run upload-coverage-coveralls"
```

Comment:
Travis does not support 2.1 so will pass it a 2.0 dockerfile with the latest tag for the MONGO_IMAGE_TAG
<img width="929" alt="screen shot 2017-02-10 at 09 06 23" src="https://cloud.githubusercontent.com/assets/1902266/22820871/873fad70-ef70-11e6-8bf6-ef9fe8bb3b1f.png">
Will raise an issue to check when we can upgrade it